web/marketing/pages/free-offers.html

```diff
@@ -13,49 +13,38 @@
             <div class="col col-heading">Expiration</div>
             <div class="col col-heading">Status</div>
             <div class="col col-heading edit-offer"></div>
-        </div>
-        <hr>
+        </div><hr>
         <div class="row offer-heading ">
             <p class="offer-type">DEFAULT&nbsp;OFFER</p>
         </div>
-        <div class="row data-row">
-            <div class="col ml-3">May Referral</div>
-            <div class="col">$50</div> 
-            <div class="col">$50</div>
-            <div class="col">48</div>
-            <div class="col">200</div>
-            <div class="col">05/01/19</div>
-            <div class="col">06/01/19</div>
-            <div class="col"><strong>Live</strong></div>
-            <div class="col col-heading edit-offer">EDIT</div>
-        </div>
-        <hr>
+        {{range .FreeCredits}}
+            {{if eq .Status 2}}
+                <div class="row data-row">
+                    <div class="col ml-3">{{.Name}}</div>
+                    <div class="col">${{.InviteeCreditInCents}}</div>
```

Comment:
We should either let user know this is a `cent` field or we do the conversion ourselves in the handler.